src/java.base/unix/classes/sun/nio/fs/UnixPath.java

```diff
@@ -811,6 +811,7 @@ public UnixPath toAbsolutePath() {
     }
 
     @Override
+    @SuppressWarnings("removal")
```

Comment:
I assume this can be removed.

Reply:
Removed in c193b3b48f56e2c581efaf212f2770c9add36a0a.